server/request_handler.go

```diff
@@ -137,3 +149,13 @@ func (r *RpcRequestHandler) processBatchRequest(client RPCProxyClient, jsonBatch
 	// Write consolidated response
 	r._writeRpcBatchResponse(response)
 }
+
+func (r *RpcRequestHandler) finishRequest() {
+	timeRequestNeeded := time.Since(r.timeStarted) // At end of request, log the time it needed
+	r.requestRecord.requestEntry.RequestDurationMs = timeRequestNeeded.Milliseconds()
+	if len(r.requestRecord.ethSendRawTxEntries) > 0 {
+		r.db.SaveRequestEntry(r.requestRecord.requestEntry)
```

Comment:
I think there should be 1 call `r.db.SaveRequest` that saves both the requestEntry as well as all the rawTxEntries, and that should be spun off into a goroutine to not block the main request.

Reply:
Done